delta/json-schema/delta-protocol.ess

```diff
@@ -20,6 +20,13 @@ namespace Lion.WebCore.Serialization
 	}
 	class SerializedNode
 	{
+		string Id;
```

Comment:
Wouldn’t it be useful to have a type alias `Id`?

Reply:
I don't think there are type aliases in JsonSchema

Reply:
But this is (C#-like) Essential code.

Reply:
... which will never be merged, as per https://github.com/LionWeb-io/specification/pull/338#discussion_r1985329757